packages/common/src/utils/arrays/random.ts

```diff
@@ -0,0 +1,13 @@
+export const ifTheSameElseDefault = <
```

Comment:
random? is this a way of creating a `utils` file without calling it that?
love the `ifTheSameElseDefault`! nice use of typescript too 😁

Reply:
🤣 

Honestly, I don't know where to put it 😢